onboard/DevcadeClient.cs

```diff
@@ -126,11 +128,35 @@ private static void Chmod(string filePath, string permissions = "700", bool recu
             notifyDownloadComplete(game);
         }
         
+        public static void reportToDatadog(DevcadeGame game) {
+            // Create a new UdpClient
+            UdpClient udpClient = new UdpClient();
+
+            // Create a new IPEndPoint for the destination IP address and port number
+            IPAddress ipAddress = IPAddress.Parse("127.0.0.1");
```

Comment:
Should this be a env var or something or do you intend to add another service on the decade machine to then send this to datadog?

Reply:
in that case this looks good to me, ill add datadog-agent to the list of installed packages  in the setup guide